docs/docs/guides/how-to-promote-content.md

```diff
@@ -0,0 +1,66 @@
+import SelectProject from './assets/promote-select-project.png';
+import PromoteViewChart from './assets/promote-view-chart.png';
+import PromoteListChart from './assets/promote-list-chart.png';
+import SucessMessage from './assets/promote-success.png';
+
+# How to promote content
+
+Promoting content enables you to copy content from one project to another, this can also be used to replicate changes from one `development project` into a `production project`. This helps keep everything consistent and makes it easier to manage changes.
```

Comment:
```suggestion
Promoting content enables you to copy content from one project to another, this can also be used to replicate changes from one `preview project` into a `production project`. This helps keep everything consistent and makes it easier to manage changes.
```

I don't know if we should use preview project here? Development project is more like "intuitive", but it's likely that the dev project is actually a preview project.